include/dba.php

```diff
@@ -91,6 +91,22 @@ public function getdb() {
 		return $this->db;
 	}
 
+	/** Returns the MySQL server version string
```

Comment:
yes this is possible but just for consistency (with the other doxygen headers in friendica) it would be nice if you could change it to
```
/**
 * @brief Returns the MySQL server version string
```